script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -942,7 +1004,9 @@ The executioner inches closer, showing off the sounds of her footsteps.
 [sha:8237bbc7031506cb6917dec017fa7882fffca091]{
 -- Page 306, Offset 34086.
 -- 「せめてあがいたらどうですか。私を殺せば一日は生き長らえる。そんなありえない希望を見ながら、無様に死ぬのが貴方にはお似合いです」
-"How about at least trying to put up a fight? If you kill me, then you'll live to see another day. You should die a miserable death while clinging to that impossible hope; it'd suit you perfectly."
+"How about at least trying to put up a fight?
+ If you kill me, you'll get to live for another whole day.
+ Dying while pitifully clinging to that vain hope would suit you ever so well."
```

Comment:
isn't it "whole another day"...?

Reply:
Can't say I've ever heard it that way around, do you have any examples?

Reply:
It's more that "whole another" is an ancient English collocation, to the point where it had an alternative spelling.

Reply:
I really don't trust raw google counting, because if you look at the hits
> another day of digging your *own whole another* day in purgatory - Typo of digging your own hole
> Erin Lives *Whole - Another day*, another no bake granola bar recipe - Not an actual instance
>  Or restart the whole Another Day chapter - Proper name
> She swallowed whole another day - Also not an actual match

you get a high incidence of not-actually-what-you-meant. ngram corpus is all real printed literature, so I trust it to be a bit cleaner data / written by people who know how to write, rather than Amazon reviewers. At any rate, "live for whole another day" does not look correct to me here. If you want I can run it by some real tame college-educated Americans for some outside perspective.

Reply:
Well, it would be "live for a whole another day", but regardless. Say it without the "a" and it probably feels more familiar in your mouth ("whole 'nother")

Reply:
That construction feels kinda cowboy-go to me, maybe we just take this one to the gpr round table as well